src/main/resources/static/resources/testRules_constants.js

```diff
@@ -0,0 +1,42 @@
+// Short usage description of test rules
+var test_rule_info = '1. Load buttons are for loading rules/events from extern files.\n' +
+                    '    It is possible to choose if you want to append or replace\n' +
+                    '    aready writen rules/events in the text areas.\n'+
+                    '      * Upload rules/events must be in json list, ex. [{},{}].\n' +
```

Comment:
Suggestion: The file content should be formatted as json objects in a json list, ex. [{Object1}, {Object2}].\n' +

Reply:
Done